packages/commonwealth/server/scripts/purgeCloudflareCache.ts

```diff
@@ -0,0 +1,37 @@
+import { config } from 'dotenv';
+import fetch from 'node-fetch';
+
+config();
+
+async function purgeCache(zoneId?: string, apiKey?: string) {
+  if (!zoneId || !apiKey) throw Error('Missing Env Vars');
+
+  const url = `https://api.cloudflare.com/client/v4/zones/${zoneId}/purge_cache`;
+  const body = {
+    purge_everything: true,
+  };
+  const headers = {
+    Authorization: `Bearer ${apiKey}`,
+    'Content-Type': 'application/json',
+  };
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: headers,
+      body: JSON.stringify(body),
+    });
+
+    const responseData = await response.json();
+    console.log('Cache purge request successful:');
```

Comment:
Use the pino logger.

Reply:
Ah nevermind, I will change it, becuase if we import the config stuff anyways it will import commonwealth/core, so it won't matter in this case

Reply:
Unresolving because I am still seeing console logs...

Reply:
Sorry, I don't know how I had missed some. Fixed them all now